bugtool/cmd/root.go

```diff
@@ -315,6 +319,39 @@ func runAll(commands []string, cmdDir string, k8sPods []string) {
 			writeCmdToFile(cmdDir, cmd, k8sPods, enableMarkdown)
 			continue
 		}
+
+		if strings.Contains(cmd, "xfrm") {
+			var output []string
+			execResult, _ := execCommand(cmd)
+			// splitting the result as an array of string
+			lines := strings.Split(execResult, "\n")
+			// list of substrings to search for in lines
+			substrings := [...]string{"auth", "enc", "aead", "auth-trunc", "comp"}
```

Comment:
Lines with `auth-trunc` will already match on `auth` and be processed before trying `auth-trunc`. This pattern should be removed (possibly kept in a comment, in case we remove `auth` some day), or, if the processing differs between `auth-trunc` and `auth`, `auth` should be adjusted accordingly.